docs/external_links.rst

```diff
@@ -0,0 +1,15 @@
+Related External Links
```

Comment:
I think a better name would be "Third-party Resources" (or something like that)

Reply:
I agree